CHANGELOG.md

```diff
@@ -1,8 +1,48 @@
+### citus v5.1.0 (May 17, 2016) ###
+
+* Adds support for using EXPLAIN on distributed queries
+
+* Improves planner performance when using tables with many shards
+
+* Adds ability to pass columns as arguments to function in UPDATEs
+
+* Introduces transaction manager for use by multi-shard commands
+
+* Adds distributed COPY to rapidly populate distributed tables
```

Comment:
Is this for 'append' then? Maybe group both of them together, and note deprecate STAGE for COPY for append partitioned tables.